src/migrations/20220316143724_payment_gateway_meta_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class RecurrentPaymentStateIndex extends AbstractMigration
+{
+    public function up()
+    {
+        if ($this->hasTable('payment_gateway_meta')) {
+            return;
+        }
+
+        $this->table('payment_gateway_meta')
+            ->addColumn('payment_gateway_id', 'integer', ['null' => false])
+            ->addColumn('key', 'string', ['null' => false, 'limit' => 255])
```

Comment:
I'd also add an index for `key` column. Since you already have the table, consider adding the index yourself too.